static/app/views/automations/components/actions/slack.tsx

```diff
@@ -71,18 +78,31 @@ export function SlackNode() {
           notes: <NotesField />,
         })}
       </OptionalRowLine>
-      <InfoBanner>
-        <Flex gap={space(0.5)}>
+      {dismissed ? null : (
+        <Alert
```

Comment:
Since we're using this in multiple places, can you make a new `DismissableAlert` component?

Reply:
you could also use `useDismissAlert()` instead of state if you want it to persist

Reply:
fixed in https://github.com/getsentry/sentry/pull/94235/commits/670989ba52c880a2e83ad40469bd9655cb5b61b3 ! i wasn't sure if it was worth pulling out into a new component since the goal was to reduce the number of custom components, but this makes it cleaner.

i think the banner is useful for people to see since it's about common problems, so i didn't want the state to persist

<img width="724" alt="Screenshot 2025-06-25 at 2 52 08 PM" src="https://github.com/user-attachments/assets/218ccaec-b53b-45ea-9275-2421becea5c7" />

Reply:
Looks good! I have no problem with new components - I think the issue with InfoBanner was that we were duplicating a UI pattern that already existed in the app (with Alert)